python/python/lance/dataset.py

```diff
@@ -499,6 +504,7 @@ def setopt(opt, val):
         setopt(builder.use_scalar_index, use_scalar_index)
         setopt(builder.fast_search, fast_search)
         setopt(builder.include_deleted_rows, include_deleted_rows)
+        setopt(builder.order_by, orderings)
```

Comment:
Could we make this match by calling it `order_by()`? I feel like that will more closely mirror the SQL keyword it's similar to.